src/pages/index.js

```diff
@@ -116,3 +263,63 @@ profileEditButton.addEventListener("click", () => {
 
 //add card button
 addNewCardButton.addEventListener("click", () => newCardPopup.open());
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "822d6207-f743-4e11-ba27-e254a062d9f1",
+    "Content-Type": "application/json",
+  },
+});
+
+let section;
+
+api.getAll().then(([initialCards, userData]) => {
+  userInfo.updateProfileImage(userData.avatar);
+  userInfo.setUserInfo({
+    title: userData.name,
+    description: userData.about,
+  });
+  section = new Section(
+    {
+      items: initialCards,
+      renderer: renderCard,
+    },
+    ".cards__list"
+  );
+  section.renderItems();
+});
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere**